### PR TITLE
Update main repo README: links to new docs + formatting

### PR DESCRIPTION
**Pull request by konrad-jamrozik**
Complementary update to:
- https://dev.azure.com/azure-sdk/internal/_git/azure-sdk-docs-eng.ms/pullrequest/796

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,14 +2,13 @@
...
 
 ## Getting started
-If you're a Microsoft employee looking for information about all of the repositories and steps for Azure SDK Libraries Releases, go to the [Azure SDK - Internal Wiki](https://aka.ms/jointhesdk). To get access to the wiki, you need to request access to the [Azure SDK Release Partners](https://aka.ms/azsdk/join/azuresdkpartners) security group. This is a MyAccess group and requests to join will need to be approved by your manager.
-
-External Contributors can read [Getting Started with OpenAPI Specifications](https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/Getting%20started%20with%20OpenAPI%20specifications.md).
 
+If you're a Microsoft employee looking for information about all of the repositories and steps for Azure SDK Libraries Releases, go to the [aka.ms/azsdk/join](https://aka.ms/azsdk/join). To get access to the wiki, you need to request access to the [Azure SDK Release Partners](https://aka.ms/azsdk/join/azuresdkpartners) security group. This is a MyAccess group and requests to join will need to be approved by your manager.
```
Comment (maririos) [inline README.md:9]:
```suggestion
If you're a Microsoft employee looking for information about all of the repositories and steps for Azure SDK Libraries Releases, go to the [aka.ms/azsdk/join](https://aka.ms/azsdk/join).
```

Comment (maririos) [reply README.md:9]: Because now the join link will point to Eng hub where people don't need to request access

**Review by maririos — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (konrad-jamrozik) [reply README.md:9]: Thank you @maririos, I was wondering what to do about this aka.ms link.

Comment (weshaggard) [reply README.md]: >https://aka.ms/azsdk/join/azuresdkpartners

This alias itself can definitely not be deleted it is used a lot however I suspect its usage here might be able to be removed as we are moving all the docs from the wiki.

**Pull request merged**